src/Common/tests/System.Net/Capability.cs

```diff
@@ -1,145 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why was this deleted from sn2?

These files are still needed by other contracts what at this point have no testing. Please leave files in place even if they are not used by sockets. I'd rather not do the work again to port things from TFS to GitHub.